### PR TITLE
perf: Remove unused es6.number.constructor & es6.function.name polyfills

### PR DESCRIPTION
**Pull request by rkunev**
Hi! I was doing a POC for Vue components built with `--target lib` and Vue CLI, but instead of using webpack it's using Rollup. Generally speaking, using Rollup as a module bundler will result in a smaller bundle size, but you can get the most out of the default Vue CLI build system by just using a better `@babel/preset-env` config, tuned for bundling components as a library.

Before:
```
$ vue-cli-service build --target lib --name vue-read-progress ./src/components/index.js

DONE  Compiled successfully in 5759ms

  File                                      Size             Gzipped

  dist/vue-read-progress.umd.min.js    18.41 KiB          7.17 KiB
  dist/vue-read-progress.umd.js        53.80 KiB          14.80 KiB
  dist/vue-read-progress.common.js     53.40 KiB          14.68 KiB

Images and other types of assets omitted.
```

After (using `useBuiltIns: false`):
```
$ vue-cli-service build --target lib --name vue-read-progress ./src/components/index.js

DONE  Compiled successfully in 4945ms

  File                                      Size             Gzipped

  dist/vue-read-progress.umd.min.js    9.40 KiB           3.79 KiB
  dist/vue-read-progress.umd.js        30.93 KiB          8.69 KiB
  dist/vue-read-progress.common.js     30.53 KiB          8.57 KiB

Images and other types of assets omitted.
```

Per the [`@vue/babel-preset-app` docs](https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/babel-preset-app#usebuiltins):

> If you are building a library or web component instead of an app, you probably want to set this to false and let the consuming app be responsible for the polyfills.

In your specific case `es6.number.constructor` & `es6.function.name` polyfills are added, because there's a prop validator with the `Number` constructor and the `VueReadProgress.name` in `src/components/index.js`. Babel is working on the safe side and includes **9 KiB** of polyfills, but you can safely omit them.

tl;dr: **51%** size reduction with no additional cost 🎉

### CODE REVIEW

**Comment by ajerez** [comment]
Good point! Thanks for all. I'll merge it now.
